src/main/resources/languages/english.yml

```diff
@@ -17,6 +17,8 @@ Messages:
     - "&6To join use &f/PrisonEscape join"
     - "&6There are already &f{PLAYERS} &6waiting"
     - "&6==================================="
+    PrisionerArrested: "&4 Prisioner &f{PLAYER} has been sent to solitary"
```

Comment:
Remove space after color code, and add color code after player name. How it is now the message will be red until the player name and white afterwards. I suppose you wanted only the player name to be white.